python/mxnet/test_utils.py

```diff
@@ -513,8 +513,7 @@ def check_symbolic_forward(sym, location, expected, rtol=1E-4, atol=None,
 
     executor = sym.bind(ctx=ctx, args=location, args_grad=args_grad_data, aux_states=aux_states)
     for g in executor.grad_arrays:
-        if g:
-            g[:] = 0
+        g[:] = 0
```

Comment:
`grad_array` is either a list of NDArray or None, cannot be list of None, then `g` only can be a ndarray. Also, the original statement `if g` is equal to `if True`